### ISSUE TITLE
Excutable Missing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#393]
```
Describe your system:
 - Operating System (be specific):Ubuntu Linux 10.10 Maverick 
 - Machine type (32-bit or 64-bit):32-bit
 - Mupen64Plus version:1.99.3
- Plugins used:

For visual artifacts, give:
- game name and CRC/MD5 hash (printed on console)
- description of the artifact (missing polygons, flashing, wrong colors,
etc)

Describe the problem: Using the instructions from the read me files I installed 
Mupen64Plus 1.99.3, but for some reason the executable is missing from 
usr/local/bin (I dont have a bin folder). (So far I only have version 1.5). I 
have already installed 1.99.3 multiple times and yet the executable is not put 
there.


Please provide any additional information below.
Mind putting the new version of Mupen64Plus in the Ubuntu Repos (Ubuntu 
Software Center) because it only has the old v1.5 in the repos. 

```

Original issue reported on code.google.com by `elementa...@gmail.com` on 16 Oct 2010 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
I can also confirm this issue with mupen 1.99.3 installer on Ubuntu 10.04 
amd64. 

Some kids on the ubuntu forum [not me! also seem to be having problems: 
http://ubuntuforums.org/showthread.php?t=1357611 ]

I've come to the same conclusion as them. 

In the setup script, it tries to install stuff in /usr/local/bin/ but nothing 
ends up there.

These are the lines in install.sh that configure that.

PREFIX="${1:-/usr/local}"
SHAREDIR="${2:-${PREFIX}/share/mupen64plus}"
BINDIR="${3:-${PREFIX}/bin}"

I suspect Ubuntu probably wants the files to be dropped in a different location 
rather than just "mkdir -p"'ing the locations but I'm not really sure where to 
be honest.

my hacking around:


timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash 
install.sh 
Installing Mupen64Plus Binary Bundle to /usr/local
install: creating directory `/usr/local/share/mupen64plus'
install: creating directory `/usr/local/share/mupen64plus/doc'
install: omitting directory `doc/emuwiki-api-doc'
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls 
/usr/local/bin/
ls: cannot access /usr/local/bin/: No such file or directory
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls 
/usr/local/lib/
libmupen64plus.so.2  libmupen64plus.so.2.0.0  python2.6  site_ruby
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash 
uninstall.sh 
Uninstalling Mupen64Plus Binary Bundle from /usr/local
rmdir: failed to remove `/usr/local/bin': No such file or directory
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ mkdir -p 
/usr/local/bin
mkdir: cannot create directory `/usr/local/bin': Permission denied
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo mkdir -p 
/usr/local/bin
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ sudo bash 
install.sh 
Installing Mupen64Plus Binary Bundle to /usr/local
install: creating directory `/usr/local/share/mupen64plus'
install: creating directory `/usr/local/share/mupen64plus/doc'
install: omitting directory `doc/emuwiki-api-doc'
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ ls 
/usr/local/bin/
timmy@speed-freed:/tmp/test/mupen64plus-bundle-linux64-1.99.3$ 

```

Original comment by `timmydob...@gmail.com` on 17 Oct 2010 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Yea I would really like this to be fixed. The old Mupen64Plus wont let me 
change my PS3 controller settings and wont even make it work half the time. 
Could the devs just update the version in the repos to 1.99.3?
```

Original comment by `elementa...@gmail.com` on 21 Oct 2010 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
ppa:sven-eckelmann/ppa-mupen64plus

And no, Debian will not get 1.99.3 and Ubuntu only fetches it from Debian.
```

Original comment by `sven@narfation.org` on 21 Oct 2010 at 11:32

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
elementartes wrotes:
> How do I find the launcher for the Graphical Interface of Mupen64Plus?

There is no graphical interface in mupen64plus 1.99.x - please see 
http://code.google.com/p/mupen64plus/wiki/ThirdPartyPlugins for some options
```

Original comment by `sven@narfation.org` on 23 Oct 2010 at 2:44

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 1.99.4.  Was caused by install script quitting after the 
'omitting directory...' message.
```

Original comment by `richard...@gmail.com` on 29 Nov 2010 at 1:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you. :D
```

Original comment by `timmydob...@gmail.com` on 4 Dec 2010 at 11:30
